source/2019-09-20-the-ember-times-issue-116.md

```diff
@@ -7,21 +7,19 @@ responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
-
-This week:
-
+Check out the new Octane Super Rentals Tutorial 🚀,
 the 1st EmberJS Meetup in Nizhny 🥳,
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Octane Super Rentals Tutorial 🚀](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Godfrey Chan @chancancode](https://github.com/chancancode) and [Vaidehi Joshi @vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing new content for the revamped Super Rentals tutorial. You can now preview the [Octane guides](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/) and check out the [tutorial](https://super-rentals-tutorial--ember-super-rentals.netlify.com/).
```

Comment:
```suggestion
[Godfrey Chan @chancancode](https://github.com/chancancode) and [Vaidehi Joshi @vaidehijoshi](https://github.com/vaidehijoshi) did some great work writing new content for the revamped Super Rentals tutorial. You can now preview the [new tutorial](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/) in the [Octane guides](https://octane-guides-preview.emberjs.com/).
```

I think _maybe_ it would be nice to link to the Super Rentals app too, but it seems hard to fit it in, and the tutorial text already links you to it.